packages/website/ts/components/top_bar/top_bar.tsx

```diff
@@ -211,15 +216,21 @@ export class TopBar extends React.Component<TopBarProps, TopBarState> {
         // TODO : Remove this once we ship portal v2
         const shouldShowPortalV2Drawer = this._isViewingPortal() && utils.shouldShowPortalV2();
         return (
-            <div style={{ ...styles.topBar, ...bottomBorderStyle, ...this.props.style, ...{ height } }} className="pb1">
-                <div className={parentClassNames} style={{ maxWidth: this.props.maxWidth }}>
-                    <div className="col col-2 sm-pl1 md-pl2 lg-pl0" style={{ paddingTop: 15 }}>
-                        <Link to={`${WebsitePaths.Home}`} className="text-decoration-none">
-                            <img src={logoUrl} height="30" />
-                        </Link>
-                    </div>
-                    <div className={`col col-${isExpandedDisplayType ? '8' : '9'} lg-hide md-hide`} />
-                    <div className={`col col-${isExpandedDisplayType ? '6' : '5'} sm-hide xs-hide`} />
+            <div
+                style={{ ...styles.topBar, ...bottomBorderStyle, ...this.props.style, ...{ height } }}
+                className="pb1 flex items-center"
```

Comment:
Yea I think it's probably better to do this responsiveness stuff by using styled components and media queries? I added a way to do this that is currently broken but fixed here https://github.com/0xProject/0x-monorepo/pull/816/files#diff-00f964471edfccc5064ec54c2d1299c1R5

I think it makes sense to do it in JS when you can't do it in CSS, like when the layout is completely different on mobile and desktop.

Reply:
How do you suggest we update the `TopBarProps` interface to best support this. I want the max-width and padding to reflect how we are laying out `LargeLayout` and `SmallLayout` in `Portal`